app/views/posts/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>List of Post</h1>
\ No newline at end of file
+<h1>All Posts</h1>
+<%= render 'partials/user', user: @user %>
+
+<% @user.posts.each_with_index do |post, index| %>
+  <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+    <div class="container recent_post">
+      <p class="post-num">Post #<%= index + 1 %></p>
+      <p class="post-title"><%= post.title %></p>
+      <small class="counters">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></small>
+
+      <% if post.five_recent_comments.length.positive? %>
```

Comment:
- [ ] This specific error is coming because you do not have the method called ```five_recent_comments``` in your posts model. KIndly ensure that you use the correct method name and it will work as expected.

![image](https://github.com/KAYEMBAJONATHAN/Blog-app-/assets/86318284/92c222b1-acd8-4a75-8287-9d0bac8dc305)